backend/tests/conftest.py

```
@@ -0,0 +1,14 @@
+import pytest
+from pathlib import Path
+import sys
+sys.path.append(str(Path(__file__).resolve().parents[1]))
+
+from database.session import engine, Base
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_database():
```

Ruff (0.8.2)

Missing return type annotation for public function `setup_database`

(ANN201)